home-assistant.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Home Assistant": {
+    "description": "Home Assistant is an open-source home automation platform running on Python 3. Track and control all devices at home and automate control.",
+    "ui": {
+      "slug": ""
+    },
+    "website": "https://home-assistant.io/",
+    "version": "1.0",
+    "containers": {
+      "home-assistant": {
+        "image": "homeassistant/home-assistant",
+        "launch_order": 1,
+        "opts": [
+          [
+            "--net=host",
```

Comment:
according to documentation for home-assistant:
"Home Assistant must be on the same network as the devices for uPnP discovery to work. If running Home Assistant in a Docker container use switch --net=host to put it on the host’s network."
So removing this option will break discovery for devices such as Google Chromecast, Philips Hue, Plex Mediaserver, etc. I tried it without --net=host, rock-on starts, but discovery doesn't work indeed. So if it is possible i think it's better to keep this option